File: share/RunScript/PurgeImages.cs
using Autodesk.Revit.DB;
using SCaddins.DestructivePurge;

public static void Main(Document doc)
{
    var images = DestructivePurgeUtilitiles.Images(doc);
    DestructivePurgeUtilitiles.RemoveElements(doc, images);    
}

File: share/RunScript/PurgeImages.lua
import ('RevitAPI', 'Autodesk.Revit.DB')
import ('SCaddins', 'SCaddins.DestructivePurge')

doc = commandData.Application.ActiveUIDocument.Document
images = DestructivePurgeUtilitiles.Images(doc);
DestructivePurgeUtilitiles.RemoveElements(doc, images);    
